vulnerabilities/importers/alpine_linux.py

```diff
@@ -90,19 +91,17 @@ def updated_advisories(self) -> Set[Advisory]:
     def _process_file(self, path) -> List[Advisory]:
         advisories = []
 
-        with open(path) as f:
-            record = yaml.safe_load(f)
-
-            if record["packages"] is None:
-                return advisories
-            validate_schema(record)
+        record = load_yaml(path)
+        if record["packages"] is None:
+            return advisories
+        validate_schema(record)
 
-            for p in record["packages"]:
-                advisories.extend(
-                    self._load_advisories(
-                        p["pkg"], record["distroversion"], record["reponame"], record["archs"],
-                    )
+        for p in record["packages"]:
+            advisories.extend(
+                self._load_advisories(
+                    p["pkg"], record["distroversion"], record["reponame"], record["archs"],
```

Comment:
Using kwargs would be more explicit than plain positional args and would show immediately about how you are mapping the external data to you model. Otherwise, this is harder to read and grok.

Reply:
Didn't saw this, I only wanted to refactor the file loading logic. 

I agree with you, this is ugly code. The name `_load_advisories` doesn't make sense either.

Reply:
yes